dist/src/main/resources/api/backup-management-api.yaml

```diff
@@ -109,6 +110,77 @@ paths:
           description: Request from gateway to broker timed out
           $ref: '#/components/responses/Error'
 
+  # Section on historic backups
+  /backup-history:
+    post:
+      summary: Takes a historic backup
+      description: Start taking a backup with the given backupId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TakeBackupHistoryRequest'
+      responses:
+        '202':
+          description: A backup has been successfully scheduled
+          $ref: '#/components/responses/TakeBackupHistoryResponse'
+        '400':
+          description: In case something is wrong with backupId, e.g. the same backup ID already exists.
+        '500':
+          description: All other errors, e.g. ES returned error response when attempting to create a snapshot.
+        '502':
+          description: Elasticsearch is not accessible, the request can be retried when it is back.
+    get:
+      summary: Lists all available historic backups
+      description: |
+        Returns a list of all available backups with their state and additional info,
+        sorted in descending order of backupId.
+      responses:
+        '200':
+          description: OK
+          $ref: '#/components/responses/HistoryBackupList'
+        '404':
+          description: Backup repository is not configured.
+        '400':
+          description: In case something is wrong with backupId, e.g. the same backup ID already exists.
+        '500':
+          description: All other errors, e.g. ES returned error response when attempting to create a snapshot.
+        '502':
+          description: Elasticsearch is not accessible, the request can be retried when it is back.
+  /backup-history/{backupId}:
+    get:
+      summary: Get information of a historic backup
+      description: A detailed information of the backup with the give backup id.
+      parameters:
+        - $ref: '#/components/parameters/BackupId'
+      responses:
+        '200':
+          description: OK
+          $ref: '#/components/responses/HistoryBackupDetail'
+        '400':
+          description: In case something is wrong with backupId, e.g. the same backup ID already exists.
```

Comment:
:x: Same here, maybe double-check all the descriptions and adjust them slightly.

Reply:
My bad, copy paste from the errors [Take Backup](https://docs.camunda.io/docs/next/self-managed/operational-guides/backup-restore/operate-tasklist-backup/#create-backup-api)